src/server.ts

```diff
@@ -0,0 +1,58 @@
+import { createServer, IncomingMessage, ServerResponse } from 'http';
+import { handler } from './lambda-handler.js';
+
+// Define the server
+const server = createServer(async (req: IncomingMessage, res: ServerResponse) => {
+	// Constructing API Gateway event
+	const url = new URL(req.url!, `http://${req.headers.host}`);
+	const apiGatewayEvent = {
+		version: '2.0',
+		routeKey: req.url!,
+		rawPath: url.pathname,
+		rawQueryString: url.searchParams.toString(),
+		headers: req.headers,
+		requestContext: {
+			accountId: '123456789012', // replace with your AWS account ID
```

Comment:
Can this be an env var instead?

Reply:
can just remove the comment. this is meant to be a dummy value